fapolicy_analyzer/glade/policy_rules_admin_page.glade

```diff
@@ -15,163 +16,267 @@
   You should have received a copy of the GNU General Public License
   along with this program.  If not, see <https://www.gnu.org/licenses/>.
 -->
-
-<!-- Generated with glade 3.38.2 -->
 <interface>
   <requires lib="gtk+" version="3.22"/>
-  <object class="GtkBox" id="policyRulesAdminPage">
+  <object class="GtkPaned" id="policyRulesAdminPage">
```

Comment:
I think the file changes here are related to PR #491 not this PR. If that's the case please revert this file.